Foody/Onboarding/Login/LoginViewController.swift

```diff
@@ -6,29 +6,112 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var loginImageView: UIImageView!
```

Comment:
Intentions are very broken. Could you please fix it ? 
Hint:  Control + I 
If you want you can select all codes with command+A and then control+I

Reply:
Still have intention problem. Please fix it.